### ISSUE TITLE
[bug] Google search next/prev shows 2 hints rather than go directly

### ISSUE THREAD

**Issue by vcfvct** [brookhong/Surfingkeys#650]
The behavior used to be: click `]]` or `[[` to go back and forth directly. Now it shows 2 hints and user has to enter another key to navigate. As shown below. 
![screen shot 2018-01-14 at 15 45 41](https://user-images.githubusercontent.com/2096033/34920545-7b4f9bd8-f942-11e7-99ca-75b8be31596e.png)
![screen shot 2018-01-14 at 15 46 04](https://user-images.githubusercontent.com/2096033/34920546-7ce91136-f942-11e7-97ad-291c5601eeee.png)

Looks like the Regex is [here](https://github.com/brookhong/Surfingkeys/blob/master/content_scripts/runtime.js#L25) and called by this [nextPage function](https://github.com/brookhong/Surfingkeys/blob/master/content_scripts/content_scripts.js#L221). 
I was trying to reproduce this in chrome console after injecting jquery 3.2.1, however i am getting `Syntax error, unrecognized expression: unsupported pseudo: css` as well as `TypeError: $(...).regex is not a function`. 

Any ideas how to get this fixed? 
Your extension is **GREAT** and super helpful. Thanks for sharing!

**Comment by vcfvct** [comment]
I just realize I have to add the below function for your private `regex()` function to work. However still cannot figure out the `:css` pseudo. 

```
(function($){
    $.fn.regex = function(pattern, fn, fn_a){
        var fn = fn || $.fn.text;
        return this.filter(function() {
            return pattern.test(fn.apply($(this), fn_a));
        });
    };
})(jQuery);
```

**Comment by vcfvct** [comment]
okay, looks like you have other secret jquery weapons. The `:css` one is  
```
    $.expr[':'].css = function(elem, pos, match) {
        var sel = match[3].split('=');
        return $(elem).css(sel[0]) == sel[1];
    };
```

So the issue is in `Google` search page, both the parent `<a>` tag and the child `span` with text `Next` are selected.

**Comment by vcfvct** [comment]
@brookhong , any good ideas how to fix this? changing selector or add some conditions to check whether targets are child/parent relations?

**Comment by brookhong** [comment]
You don't have to add those jquery customization functions manually, just switch to Surfingkeys context from console.

![image](https://user-images.githubusercontent.com/288207/34971521-0f8796b0-fab6-11e7-8653-403662a2a246.png)

Below change should fix the issue, will release in next version.

    function nextPage() {
        var nextLinks = $('a:visible, button:visible, *:visible:css(cursor=pointer)').regex(runtime.conf.nextLinkRegex);
        nextLinks = $(filterOverlapElements(nextLinks.toArray()));
        if (nextLinks.length) {
            clickOn(nextLinks);
            return true;
        } else {
            return walkPageUrl(1);
        }
    }

**Comment by vcfvct** [comment]
Oh, great! Good to know I can switch context to extension directly in the console!
Y, i believe this should fix the issue. @brookhong Indeed you have a lot of secret weapons(like the functions I injected above and this `filterOverlapElements`) 😃. 
Shall I close this issue? 
Thanks man!

**Comment by brookhong** [comment]
Fixed in 0.9.25.
